CONTRIBUTING.md

```diff
@@ -100,20 +102,33 @@ ensure the 32bit and 64bit versions are both installed into C:\openjdk
 Fixes: #1234
 ```
 
-All changes should be made to a personal fork of AdoptOpenJDK/infrastructure for making changes.
+All changes should be made to a personal fork of adoptium/infrastructure for making changes so the following standard GitHub workflow should be used.
 
 1. Fork this repository
 1. Create a branch off your fork
 1. Make the change
 1. Test it (see below)
 1. Submit a Pull Request
 
-Only reviewers in the
-[infrastructure](https://github.com/orgs/AdoptOpenJDK/teams/infrastructure)
-team have permission to merge requests for this repo, so if you feel your PR
+If you are new to git and GitHub and the above makes no sense to you then [this primer may be useful](http://sxatech.blogspot.com/2021/12/git.html).
+
+Only [committers to the adoptium.temurin project](https://projects.eclipse.org/projects/adoptium.temurin/who)
+have permission to merge requests for this repo, so if you feel your PR
 is not getting enough attention, let one the team know via the
 `#infrastructure` slack channel
 
+If you are adding any downloads into the playbooks, please ensure this is
+done over https or another secure channel, and that a GPG checksum of the
+file is verified if possible, and if not a SHA checksum is performed after
+the download has been performed. There are various examples of this in the
+playbooks:
+
+- [gmake role](https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gmake/tasks/main.yml) (GPG verification using [package_signature_verification.sh](https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/Supporting_Scripts/package_signature_verification.sh)
+- [NVidia_Cuda_Toolkit role](https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml) which performs a SHA256 check of the download
+- The [gcc_11 role](https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gcc_11/tasks/main.yml) is an example of SHA checks when there are multiple downloads for each architecture. It uses checksums stored in a [separate variables file](https://github.com/adoptium/infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/gcc_11/vars/main.yml)
+ 
+Where possible, if you are modifying a playbook to add somethign new, please also add information saying what it is needed for (it may be useful to link back to a related PR or issue)
```

Comment:
```suggestion
Where possible, if you are modifying a playbook to add something new, please also add information saying what it is needed for (it may be useful to link back to a related PR or issue)
```